package.json

```diff
@@ -19,7 +22,8 @@
   },
   "dependencies": {
     "discord.js": "~12.5.3",
-    "puppeteer": "~10.1.0"
+    "puppeteer": "^10.1.0",
+    "puppeteer-core": "^10.1.0"
```

Comment:
we are not going to use versions over 10.1 so add ~ to keep the patch of the version